src/index.ts

```diff
@@ -81,9 +82,12 @@ export function viteSvgToWebfont<T extends GeneratedFontTypes = GeneratedFontTyp
             }
             await generate();
             if (isBuild) {
+                const cssFontsUrl = processedOptions.cssFontsUrl ? relative(processedOptions.dest, processedOptions.cssFontsUrl) : 'assets';
                 const emitted = processedOptions.types.map<[T, string]>(type => [
                     type,
-                    `/${this.getFileName(this.emitFile({ type: 'asset', fileName: `assets/${processedOptions.fontName}-${guid()}.${type}`, source: generatedFonts?.[type] }))}`,
+                    `/${this.getFileName(
+                        this.emitFile({ type: 'asset', fileName: `assets/${processedOptions.fontName}-${guid()}.${type}`, source: generatedFonts?.[type] }),
+                    ).replace('assets', cssFontsUrl)}`,
```

Comment:
would this work also?
```suggestion
                        this.emitFile({ type: 'asset', fileName: `${cssFontsUrl}/${processedOptions.fontName}-${guid()}.${type}`, source: generatedFonts?.[type] }),
                    )`,
```

Reply:
I tried, but unfortunately, it appears to be unsuccessful.

Reply:
Doesn't this mean we emit the file to 1 location (`assets`), and then import it from another location (`cssFontsUrl`)?

Reply:
Yes, indeed. It may be because I specified `rollupOptions.build.out`, so the emitted files will be placed in the `assets` folder under the out directory.

Reply:
Would this work when not setting that setting as you have?
Would you be willing to fix this if not?